sdk/core/core-client/test/browser/serializer.spec.ts

```diff
@@ -1,43 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-
-import { Mapper, createSerializer } from "../../src";
-import { assert } from "chai";
-
-describe("Serializer (browser specific)", function () {
-  describe("serialize", function () {
-    it("Should accept File objects as Blobs", function () {
```

Comment:
this file was removed, but I don't see where these tests were added back for browser?

Reply:
Added back file